csrc/scheduler/mma_utils.cpp

```diff
@@ -1231,27 +1231,12 @@ RolesMapOpt getTensorRoles(
     }
   }
 
-  // NOTE: sort output roles in descending order by uses() size, and
-  //  if equal then by name() to ensure the stable ordering of tensor
-  //  views in collections assigned to the supported roles
-  std::sort(storage.begin(), storage.end(), [](TensorView* a, TensorView* b) {
-    return (a->uses().size() == b->uses().size())
-        ? (a->name() < b->name())
-        : (a->uses().size() > b->uses().size());
-  });
-
```

Comment:
I realized this is redundant since we now do this for all roles, not just outputs.